runtime/src/main/java/org/corfudb/runtime/ILivenessUpdater.java

```diff
@@ -0,0 +1,6 @@
+package org.corfudb.runtime;
+
+public interface ILivenessUpdater {
```

Comment:
Don't use "I" prefix

Reply:
I tried to follow the interface naming convention based on our repo. Like the IHoleFillPolicy, IClient, IServerRouter, etc. Changing this might hurt the pattern for interfaces in the repo right?

Reply:
That's old naming, very old, it's just not fixed, please don't follow it

Reply:
It could be old naming style but changing the interface names here might affect the pattern followed in the repo and create inconsistency.